include/tachimawari/control/packet/protocol_1/instruction/sync_write_packet.hpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2021 Ichiro ITS
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#ifndef TACHIMAWARI__CONTROL__PACKET__PROTOCOL_1__INSTRUCTION__SYNC_WRITE_PACKET_HPP_
+#define TACHIMAWARI__CONTROL__PACKET__PROTOCOL_1__INSTRUCTION__SYNC_WRITE_PACKET_HPP_
+
+#include <string>
+#include <vector>
+
+#include "tachimawari/control/packet/protocol_1/model/packet.hpp"
+#include "tachimawari/joint/model/joint.hpp"
+#include "tachimawari/joint/protocol_1/mx28_address.hpp"
+
+namespace tachimawari
+{
+
+namespace control
+{
+
+namespace packet
+{
+
+namespace protocol_1
+{
+
+class SyncWritePacket : public Packet
+{
+public:
+  SyncWritePacket();
+
+  void create(
+    const std::vector<tachimawari::joint::Joint> & joints,
+    const uint8_t & starting_address =
+    tachimawari::joint::protocol_1::MX28Address::GOAL_POSITION_L);
```

Comment:
The `MX28Address`is an enum class in the `tachimawari::joint` namespace, as we discussed days ago, the complete namespace must be typed because the `sync_write_packet` class is placed in the different parent namespace. Or, did you mean why the `MX28Address` enum class is placed on the different parent namespace?

Reply:
ooh right, just realized about that, it's `tachimawari::control::packet::protocol_1` against `tachimawari::joint::protocol_1`. ok looks good.

Reply:
done